uaaextras/webapp.py

```diff
@@ -325,6 +317,19 @@ def set_headers(response):
     # make sure our base url doesn't have a trailing slash as UAA will flip out
     app.config['UAA_BASE_URL'] = app.config['UAA_BASE_URL'].rstrip('/')
 
+    app.config['APP_ROOT'] = os.path.dirname(os.path.abspath(__file__))
+    app.config['APP_STATIC'] = os.path.join(app.config['APP_ROOT'], 'static')
+
+    # Load valid list of .gov domains
+    # download current-federal.csv from
+    #   https://github.com/GSA/data/tree/gh-pages/dotgov-domains
+    # and place into 'static' dir
+    fed_gov_csv = open(os.path.join(app.config['APP_STATIC'], 'current-federal.csv'), newline='')
+    domain_regex_list = ['\.mil$', '\.fed\.us$']
+    for row in csv.reader(fed_gov_csv):
+        domain_regex_list.append(row[0].strip().rstrip().replace('.', '\.') + '$')
```

Comment:
Why does this need a regex at all here? Why not something like:

    app.config['VALID_DOMAINS'] = ['.mil']
    with open(os.path.join(APP_STATIC, app.config['VALID_DOMAIN_LIST'])) as fh:
        for row in csv.reader(fh):
            app.config['VALID_DOMAINS'].append(row[0].lower())